dist/react-jsonschema-form.js

```diff
@@ -0,0 +1,5 @@
+r}function I(e,t){return e===t||e!==e&&t!==t}function N(e){var t=R(e)?le.call(e):"";return t==Z||t==V}function R(e){var t="undefined"==typeof e?"undefined":U(e);return!!e&&("object"==t||"function"==t)}function D(e){return!!e&&"object"==("undefined"==typeof e?"undefined":U(e))}function q(e){return"symbol"==("undefined"==typeof e?"undefined":U(e))||D(e)&&le.call(e)==J}function W(e){return null==e?"":j(e)}function $(e){return ge(e)?r(e,F):q(e)?[e]:x(ve(e))}var U="function"==typeof Symbol&&"symbol"==typeof Symbol.iterator?function(e){return typeof e}:function(e){return e&&"function"==typeof Symbol&&e.constructor===Symbol&&e!==Symbol.prototype?"symbol":typeof e},B="Expected a function",L="__lodash_hash_undefined__",z=1/0,Z="[object Function]",V="[object GeneratorFunction]",J="[object Symbol]",H=/^\./,G=/[^.[\]]+|\[(?:(-?\d+(?:\.\d+)?)|(["'])((?:(?!\2)[^\\]|\\.)*?)\2)\]|(?=(?:\.|\[\])(?:\.|\[\]|$))/g,K=/[\\^$.*+?()[\]{}|]/g,X=/\\(\\)?/g,Y=/^\[object .+?Constructor\]$/,Q="object"==("undefined"==typeof t?"undefined":U(t))&&t&&t.Object===Object&&t,ee="object"==("undefined"==typeof self?"undefined":U(self))&&self&&self.Object===Object&&self,te=Q||ee||Function("return this")(),re=Array.prototype,ne=Function.prototype,oe=Object.prototype,ae=te["__core-js_shared__"],ie=function(){var e=/[^.]+$/.exec(ae&&ae.keys&&ae.keys.IE_PROTO||"");return e?"Symbol(src)_1."+e:""}(),ue=ne.toString,se=oe.hasOwnProperty,le=oe.toString,ce=RegExp("^"+ue.call(se).replace(K,"\\$&").replace(/hasOwnProperty|(func
```

Comment:
Dist files should not be versioned in this repository. They're generated locally before being published to npm.